README.md

```diff
@@ -7,7 +7,7 @@
 ## Getting started
 **1. Add the build system, and the wanted sources to your manifest.**
 
-Find your manifest file. Check `${ANDROID_BUILD_TOP}/.repo/manifest/`
+Find your manifest file. Check `${ANDROID_BUILD_TOP}/.repo/manifest/`. There may be a linked document in `${ANDROID_BUILD_TOP}/.repo/manifest.xml` that points to another location, such as `${ANDROID_BUILD_TOP}/.repo/manifests/default.xml`. The file you should edit should not be blank and should have several hundred lines.
```

Comment:
Different trees use different manifest configurations.  E.g. stock aosp has one giant manifest xml file (`.repo/manifests/default.xml`); Lineage uses a combination of `.repo/manifests/default.xml` and `.repo/local_manifests` (for device-specific projects that gets populated when you run `breakfast`); CodeAurora uses `.repo/manifests/${CAFTAG}.xml`.

Giving one set of instructions for everyone doesn't seem appropriate.  However, I do like the idea of giving an example of how to update the manifest in a stock AOSP tree (this is what the PR essentially does and would just need a minor edit to present it as an example).

Note that we used to use a combination of `.repo/manifests` and `.repo/local_manifests` for our in-house development, but switched to a single manifest project.  That has worked much better for us (devs no longer keep forgetting to update their local_manifests project).